openupgrade_scripts/scripts/product/18.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,109 @@
+---Models in module 'product'---
+new model product.combo
+new model product.combo.item
+
+# DONE: rename models and fields from point_of_sale models if installed
+
+new model update.product.attribute.value [transient]
+
+# NOTHING TO DO: new functionality
+
+---Fields in module 'product'---
+product      / product.attribute        / active (boolean)              : NEW hasdefault: default
+
+# DONE: added in pre-migration
+
+product      / product.attribute        / template_value_ids (one2many) : NEW relation: product.template.attribute.value
+product      / product.attribute.value  / active (boolean)              : NEW hasdefault: default
+
+# DONE: added in pre-migration
+
+product      / product.category         / message_follower_ids (one2many): NEW relation: mail.followers
+product      / product.category         / message_ids (one2many)        : NEW relation: mail.message
+product      / product.combo            / combo_item_ids (one2many)     : NEW relation: product.combo.item
+product      / product.combo            / company_id (many2one)         : NEW relation: res.company
+product      / product.combo            / name (char)                   : NEW required
+product      / product.combo            / sequence (integer)            : NEW hasdefault: default
+product      / product.combo.item       / combo_id (many2one)           : NEW relation: product.combo, required
+product      / product.combo.item       / company_id (many2one)         : NEW relation: res.company, isrelated: related, stored
+product      / product.combo.item       / extra_price (float)           : NEW hasdefault: default
+product      / product.combo.item       / product_id (many2one)         : NEW relation: product.product, required
+product      / product.document         / _order                        : _order is now 'sequence, name' ('name')
+
+# NOTHING TO DO
+
+product      / product.document         / sequence (integer)            : NEW hasdefault: default
+
+# DONE: create sequence matching name
+
+product      / product.pricelist        / _order                        : _order is now 'sequence, id, name' ('sequence asc, id asc')
```

Comment:
This is changing the pricelist order, so we should assign new `sequence` values to preserve the previous one.

Reply:
I don't see how this change impacts order out of joins given asc is the default and ids are unique?

Reply:
done, the others too